frontend/src/components/Review/PropertyInfo.tsx

```diff
@@ -1,28 +1,112 @@
-import React, { ReactElement } from 'react';
+import React, { ReactElement, useState, useEffect } from 'react';
+import { get } from '../../utils/call';
 import Grid from '@material-ui/core/Grid';
-import { Box, Typography, List, ListItem, ListItemText } from '@material-ui/core';
+import { makeStyles, Box, Typography, List, Link, Card, CardContent } from '@material-ui/core';
+import { CardData } from '../../App';
+import { Link as RouterLink } from 'react-router-dom';
+import { colors } from '../../colors';
+import HeartRating from '../utils/HeartRating';
+import { ReviewWithId, ApartmentWithId } from '../../../../common/types/db-types';
 
 type Props = {
-  readonly info: readonly string[];
+  readonly info: CardData[];
   readonly title: string;
 };
 
-export default function PropertyInfo({ info, title }: Props): ReactElement {
+type CardProps = {
+  readonly buildingData: ApartmentWithId;
+  readonly numReviews: number;
+  readonly company?: string;
+};
+
+const useStyles = makeStyles({
+  aptNameTxt: {
+    fontWeight: 700,
+  },
+  card: {
+    borderRadius: '10px',
+    backgroundColor: colors.red6,
+  },
+  reviewNum: {
+    fontWeight: 700,
+    marginLeft: '10px',
+  },
+});
+
+const PropertyCard = ({ buildingData, numReviews, company }: CardProps): ReactElement => {
+  const classes = useStyles();
+  const { id, name, address } = buildingData;
+  const [reviewData, setReviewData] = useState<ReviewWithId[]>([]);
+
+  useEffect(() => {
+    get<ReviewWithId[]>(`/review/aptId/${id}`, {
+      callback: setReviewData,
+    });
+  }, [id]);
+
+  const getAverageRating = (reviewData: ReviewWithId[]) =>
+    reviewData.reduce(
+      (currSum, { overallRating }) => (overallRating > 0 ? currSum + overallRating : currSum),
+      0
+    ) / reviewData.length;
+
+  return (
+    <Card className={classes.card}>
```

Comment:
Check the styling here on `PropertyCard`: why is there a huge gap on the page?
<img width="471" alt="Screen Shot 2022-05-03 at 4 19 32 PM" src="https://user-images.githubusercontent.com/57203639/166559415-99163118-5324-4381-a1f9-551585936508.png">